### ISSUE TITLE
How to setup sensitivity?

### ISSUE THREAD

**Issue by YuriFernandes150** [marvinbelfort/mx002_linux_driver#3]
Hello! First of all, great work on this!
I need to setup the sensitivity so it wont be so high, is there a way to do that? Im using krita, and whenever I even hover the pen over the tablet, the brush already activates, as if I'm touching the tablet. Its a very dim trace.
![image](https://github.com/marvinbelfort/mx002_linux_driver/assets/16628285/705a8e11-fc4a-41cf-a87b-e51d88c6138b)

https://github.com/marvinbelfort/mx002_linux_driver/assets/16628285/bcaf3293-f126-4dca-afd7-7e8720a3043c

I did some testing o windows and it works fine there

**Comment by marvinbelfort** [comment]
When one changes the setting to work with the entire tablet space, the hardware stops accurately detecting whether or not there is a touch. I found a distance that worked for me and hardcoded it. I believe it is possible to create a command-line option to configure this.

**Comment by YuriFernandes150** [comment]
> When one changes the setting to work with the entire tablet space, the hardware stops accurately detecting whether or not there is a touch. I found a distance that worked for me and hardcoded it. I believe it is possible to create a command-line option to configure this.

Yes, I've had to do the same as you, and hardcode it. I actually redid most of the code to work with proper constants, since the values were all hardcoded. Im thinking of making a simple interface and a json for configs.

**Comment by marvinbelfort** [comment]
I am already working on a configuration system based on TOML.

**Comment by YuriFernandes150** [comment]
> I am already working on a configuration system based on TOML.

Well, In that case, I'll just wait for you then. Thx for the work!

If its not much to ask, try to include a simple GUI with it.

**Comment by BrannonKing** [comment]
To get the sensitivity to work right on mine (a Gotop):
```diff
     fn normalize_pressure(raw_pressure: i32) -> i32 {
-        let proximity_threshold = 600; // Adjust for proximity sensitivity
-        let strength_scaling = 2; // Adjust for strength of the press
+        let proximity_threshold = 500; // Adjust for proximity sensitivity
+        let strength_scaling = 16; // Adjust for strength of the press
     
-        match 1740 - raw_pressure {
+        match 1950 - raw_pressure {
             x if x <= proximity_threshold => 0,
-            x => x * strength_scaling, // Scale the pressure for stronger presses
+            x => (x-proximity_threshold) * strength_scaling, // Scale the pressure for stronger presses
         }
     }
 ```
Notice the subtraction of proximity_threshold on the last changed line. Krita actually has a tool in its settings for testing the pen pressure. If you can get it to show a pressure range from 1 to 99 and not easily lose connection when near 10% -- that's ideal.

The 1740 number -- was that a magic number or does that come from the raw pressure spec somehow? It's just strange that raw-pressure is inverted.

**Comment by YuriFernandes150** [comment]
> I am already working on a configuration system based on TOML.

Any news on this? or is this project abandoned?
